### PR TITLE
HDDS-2804. tools/TestTools.md

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

doc translation

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2804

## How was this patch tested?

hugo server -D

### CODE REVIEW

File: hadoop-hdds/docs/content/tools/TestTools.zh.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+测试是开发分布式系统中最重要的部分，我们提供了以下类型的测试。
+
+本页面给出了 Ozone 自身的测试工具。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/tools/TestTools.zh.md:24]: 自身=》自带

File: hadoop-hdds/docs/content/tools/TestTools.zh.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
+---
+title: "测试工具"
+summary: Ozone 提供了负载生成、分片测试、验收测试等多种测试工具。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/tools/TestTools.zh.md:3]: 分片测试 （partitioning test）is not very clear to me。

File: hadoop-hdds/docs/content/tools/TestTools.zh.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+## 单元测试
+
+和每个 java 项目一样，我们的每个项目都包含传统的单元测试。
+
+## 集成测试（JUnit）
+
+传统的单元测试只能测试一个单元，但我们也有更高层次的单元测试。它们使用 `MiniOzoneCluster` 辅助方法在单元测试中启动守护进程（SCM、OM、数据节点）。
+
+从 maven 或 java 的角度来看，集成测试也只是普通的单元测试而已（使用了 JUnit 库），但为了将它们分开（同时解决一些依赖问题），我们将集成测试移到了 `hadoop-ozone/integration-test` 目录下。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/tools/TestTools.zh.md:36]: 但为了解决一些依赖问题，我们将它们分开放在集成测试 `hadoop-ozone/integration-test` 目录下。

File: hadoop-hdds/docs/content/tools/TestTools.zh.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+冒烟测试包含在发行包中（`./smoketest`），但 robot 文件只定义了运行命令行然后检查输出的测试。
+
+为了在不同环境（docker-compose、kubernetes）下运行冒烟测试，你需要定义如何启动容器，然后在正确的容器中执行正确的测试。
+
+这部分的测试包含在 `compose` 目录中（查看 `./compose/*/test.sh` 或者 `./compose/test-all.sh`）。
+
+例如，一种测试分发包的简单方法是：
+
+```
+cd compose/ozonze
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/tools/TestTools.zh.md:51]: ozonze=》ozone，please fix the typo in English version as well

File: hadoop-hdds/docs/content/tools/TestTools.zh.md (hunk 2 above)
Comment (iamabug) [reply hadoop-hdds/docs/content/tools/TestTools.zh.md]: According to the context, I would say it refers to network partition.

**Comment by xiaoyuyao** [comment]
Thanks @iamabug for the contribution and @adoroszlai for trigger the new CI check.

**Pull request merged**